crates/pop-parachains/README.md

```diff
@@ -0,0 +1,78 @@
+# pop-parachains
+
+A crate for generating, building and running parachains and pallets. 
+
+## Usage
+
+Generate a new Parachain:
```

Comment:
Same as the comment in `pop-contracts/README.md`, we should be consistent with capital letters for the words smart contracts and parachains throughout the README's. I suggest we use the version with capitals